rpc/rpcserver/server.go

```diff
@@ -796,18 +796,64 @@ func (s *walletServer) SweepAccount(ctx context.Context, req *pb.SweepAccountReq
 		return nil, translateError(err)
 	}
 
-	var txBuf bytes.Buffer
-	txBuf.Grow(tx.Tx.SerializeSize())
-	err = tx.Tx.Serialize(&txBuf)
-	if err != nil {
-		return nil, translateError(err)
+	sweepTxs := make([]*txauthor.AuthoredTx, 0)
```

Comment:
No reason to `make` a slice with zero initial length.  Use `var` to initialize the zero value instead.